csharp/Adapter/Microsoft.Spark.CSharp/Configuration/ConfigurationService.cs

```diff
@@ -21,6 +21,8 @@ internal class ConfigurationService : IConfigurationService
         public const string CSharpWorkerPathSettingKey = "CSharpWorkerPath";
         public const string CSharpBackendPortNumberSettingKey = "CSharpBackendPortNumber";
         public const string CSharpSocketTypeEnvName = "spark.mobius.CSharp.socketType";
+        public const string CSharpWorkerReadBufferSizeEnvName = "spark.mobius.CSharpWorker.readBuffer";
```

Comment:
Can we use spark.mobius.CSharpWorker.readBufferSize and spark.mobius.CSharpWorker.writeBufferSize?